docs/source/jit.rst

```diff
@@ -531,13 +531,251 @@ Python-defined Constants
 Debugging
 ~~~~~~~~~
 
-Print things
+Interpreting Graphs
+    TorchScript uses a static single assignment (SSA) intermediate representation
+    (IR) to represent computation. The instructions in this format consist of
+    ATen operators and other primitive operators, including control flow
+    operators for loops and conditionals. As an example::
 
-Use ``USE_PYTHON=0`` to debug in normal python mode
+        @torch.jit.script
+        def foo(len : int) -> torch.Tensor:
+          rv = torch.zeros(3, 4)
+          for i in range(len):
+            if i < 10:
+                rv = rv - 1.0
+            else:
+                rv = rv + 1.0
+          return rv
+
+        print(foo.graph)
+
+    A ``ScriptModule`` with a single ``forward`` method will have an attribute
+    ``graph``, which you can use to inspect the IR representing the computation.
+    If the ScriptModule has more than one method, you will need to access
+    ``.graph`` on the method itself and not the module. We can inspect the
+    graph of a method named ``foo`` on a ScriptModule by accessing ``.foo.graph``.
```

Comment:
It's a little confusion to mention a method named ``foo`` on a scriptmodule but above `foo` is the scriptmodule. Maybe rename ``foo`` in this explanation to ``some_method`` or ``bar`` ?

Reply:
Lol, good point